apps/src/componentLibrary/tags/_Tag.tsx

```diff
@@ -72,6 +72,7 @@ const Tag: React.FunctionComponent<TagProps> = ({
       )}
     >
       <div
+        tabIndex={0} // eslint-disable-line jsx-a11y/no-noninteractive-tabindex
```

Comment:
Definitely open to feedback here but I needed this to maintain parity for keyboard users being able to tab over this component and see the tool tip. If there's a better way of doing this or if this should be conditional based on a prop, let me know!

Reply:
Using tooltips on non-interactive elements is discouraged, but this seems to be the best solution without redesigning this part of the card! 😅

Reply:
Thanks for updating this!
Yeah, adding tabIndex might not be a best solution overall, but is probably the best solution in current situation.

Once `Tooltip` component will be ready I'll probably be coming back to this component and updating it, will also take a fresh look on `Tags` overall and maybe rework it slightly

Reply:
nit: making changes into Design System components also requires manually update the corresponding [CHANGELOG.md](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/componentLibrary/tags/CHANGELOG.md), please add some small description there (`0.2.0` should be a good version for this update)

Reply:
done!